0-ad-history-mod-package/simulation/templates/units/athen/hero_themistocles.xml

```diff
@@ -8,6 +8,13 @@
     <GenericName>Themistocles</GenericName>
     <SpecificName>Themistoklēs</SpecificName>
     <Icon>units/athen/hero_themistocles.png</Icon>
+    <History>
+        Themistocles was an Athenian politician and general in the 5th century BC. He was known for his populist and democratic political beliefs, which were supported by the lower classes and generally opposed by Athens' higher society. 
+        Themistocles was of the opinion that “He who commands the sea has command of everything” (quote by Themistocles): He convinced the Athenians to expand the navy by building 200 triremes, at the time, making it the most powerful fleet of Greece without any competition.
+        This would prove vital for the outcome of the second Persian invasion in 480 BC, without a doubt Themistocles' biggest achievement. Although he was not the chief commander of the Greek-allied fleet, he is credited with being the main strategist. After the Greeks were forced to retreat from their first battle with the Persians, Themistocles decided to evacuate Athens and led the fleet to the island of Salamis, waiting for the Persians to attack. The two fleets finally clashed at Salamis' strait. There, the Persian ships, larger in size and (more than double as large in) numbers, had a hard time maneuvering in the narrow space, and the smaller, more agile Greek ships emerged victorious. The Persian land army was defeated a year later.
+        Even though the Greeks had successfully defended their homeland, many cities, including Athens, had been completely destroyed and burned down. To avoid that happening again, Themistocles had a wall built around Athens and proposed the idea of building long walls to protect Athens and the passage to its port, Piraeus. These walls would not be built until 20 years later; however, they eventually turned out to be a core point of Athens' strategy in the wars to come.
```

Comment:
```suggestion
        Even though the Greeks had successfully defended their homeland, many cities, including Athens, had been completely destroyed. To avoid that happening again, Themistocles had a wall built around Athens and proposed the idea of building long walls to protect Athens and the passage to its port, Piraeus. These walls would not be built until 20 years later; but, they turned out to be a core point of Athens' strategy in the wars to come.
```